variables.tf

```diff
@@ -56,6 +56,11 @@ variable "password_policy_require_symbols" {
   default     = false
 }
 
+variable "password_policy_temporary_password_validity_days" {
+  description = "A container for information about the user pool password policy."
+  default     = 7
```

Comment:
Isn't 7 days a bit long? What's the background?

What about 1 day instead?

Reply:
This is the default value from AWS. Setting it to `7` allows us to stay backwards-compatible with existing users of the module.

Reply:
Ah, in that case it sounds fine :D